website/integrations/services/fortigate-ssl/index.md

```diff
@@ -0,0 +1,239 @@
+    -   Service Provider Binding = POST
+    -   Audience = http://service.company/remote/saml/metadata/
+    -   Signing certificate = ak.cert
+    -   Verification Certificate = Should already be filled with the certificate from the metadata you uploaded.
+    -   Property mapping:
+        -   authentik default SAML Mapping: Username
+        -   authentik default SAML Mapping: Groups
+    -   Named Property Mapping: Empty (------)
+    -   Assertion valid not before = minutes=5
+    -   Assertion valid not on or after = minutes=5
+    -   Session valid not on or after = (Set how long you want the user's session to be valid)
+    -   Default relay state = empty
+    -   Digest algorithm = sha256
+    -   Signature algorithm = sha256
+
+## Application section
+
+Lets create the application and link it to the provider
+
+### Create user group
+
+This is the user group that you matched on in the FortiGate "firewall group" above.
+
+-   Go to Directory -> Groups
+-   Click Create
+-   Name = `ak.user.group`
+-   Open ak.user.group and add the users whom should have access to the sslvpn
+-   save the group
+
+### Create the application
+
+> [!NOTE]
+> The Launch URL = blank://blank will prevent authentik from displaying it on the user's login page in authentik.
+
+-   Go to Applications -> Applications
+-   Name = What ever you fancy e.g. FGT-SSLVPN
+-   Slug = app.slug.name
+-   Group = empty (------)
+-   Provider = The provider you created before e.g. "FGT SSL SAML Provider"
+-   Backchannel Provider = empty (-----)
+-   Policy engine mode = any
+-   Launch URL = blank://blank
+-   Open in new tab = disabled
+-   icon = None
+-   Publisher = None
+-   Description = None
+-   Save
+
+### Limiting the access based on authentik group
+
+-   Open the application again
+-   Click on "Policy / Group / User Binding"
+-   Click bind existing policy
+-   Click on the "TAB" Group at the top
+-   Click on Select an object next to Group
+-   Chose ak.user.group
+-   Make sure enabled is chosen
+-   Order = 10
+-   Timeout = 30
+-   Failure result = Don't pass
+-   Click create
+
+You should now be able to login by selecting SSO login either on the portal or in FortiClient, depending on your portal configuration.
+
+> [!NOTE]
+> If you are using FortiClient remember to set the sslvpn profile to use single-signon either creating a manual profile or edit the profile in your EMS.
+
+## Troubleshooting
+
+These are just suggestions of what **could** be the cause of an issue and how to enable debug on the FortiGate
+
+> [!CAUTION]
+> Debugging can generate heavy load on a FortiGate firewall, so make sure your firewall is not already struggling with performance before you enable debugging, and remember to disabled it again when you are done.
+>
+> You can disable the debug with these commands.
+> `diag debug disable` > `diag debug reset`
+
+### Enabling debug output
+
+Before you can see any output you need to enable the debug
```

Comment:
```suggestion
Before you can see any output you need to enable the debug mode.
```

Reply:
I can't see how I can make a comment for line 194, but I would prefer that each debug command is on a new line. I am not sure if this is just because of the formatting in the diff here, or it will actually be like that when the guide is done.

> You can disable the debug with these commands.
> `diag debug disable`
> `diag debug reset`

ps. As you can probably tell, I am not used to working with commits on github (or markdown that others should read), hence why nicedevil007 make the PR for both of us. If there are any question you have asked that I have missed, feel free to throw a url or just ask me on Discord (Prestien), and I'll do my best to provide an answer.

Reply:
I agree with @DragonWin here, useuall you copy/paste those commands into the fortigates CLI (this one has a slightly different behaviour than a all known Bash CLI). I would prefer the 2 line part.